cypress/page/iframe.page.ts

```diff
@@ -0,0 +1,28 @@
+class IframePage {
+    private pageURL: string;
+    private cssBtn: string;
+    private iframePageTitle: string;
+    private cssPageURL: string;
+
+
+    constructor() {
+        this.pageURL = "https://www.w3schools.com/html/html_iframe.asp";
+        this.cssPageURL = "https://www.w3schools.com/css/default.asp";
+        this.cssBtn = "#topnav a[title='CSS Tutorial']";
+        this.iframePageTitle = "iframe[title='W3Schools HTML Tutorial']";
+    }
+    visit() {
+        cy.visit(this.pageURL);
+    }
+
+    getFrameTitle() {
+        return cy.iframe(this.iframePageTitle).find("h1");
+    }
+
+    goToCssPageInFrame() {
+        cy.iframe(this.iframePageTitle).find(this.cssBtn).click();
+    cy.frameLoaded(this.iframePageTitle, {url: this.cssPageURL});
```

Comment:
fix the indentation

Reply:
all done @holgiosalos  . In the terminal appear an error about CSS block . I was thinking if the URL is creating that problem.